tasks/configure_repos.yml

```diff
@@ -16,9 +16,7 @@
   no_log: true
 
 - name: Check if repo is already initialized
-  command: restic -r {{ item.url | trim | quote }} snapshots
-  environment:
-    RESTIC_PASSWORD: "{{ item.password | trim | quote }}"
```

Comment:
Why this is removed?

Reply:
Because the command was replaced with the wrapper script, the environment is not necessary anymore.

This makes sure the repo initialization check is performed with the correct repository (when working with multiple configs) and that Restic has all the credentials it needs